src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/fordel/andelsmessig/FordelMålbeløpPrAndel.java

```diff
@@ -109,8 +109,9 @@ private void settRegelSporing(Map<String, Object> resultater, FordelAndelModellM
 
 	private FordelAndelModell opprettNyFordeltAndel(FordelAndelModellMellomregning andelÅFordeleTil, BigDecimal beløpSomSkalFordeles, Inntektskategori beløpetsInntektskategori) {
 		var erInntektskategoriLikEksisterende = andelÅFordeleTil.getInputAndel().getInntektskategori().equals(beløpetsInntektskategori);
-		if (erInntektskategoriLikEksisterende) {
-			return opprettAndelFraEksisterende(andelÅFordeleTil, beløpSomSkalFordeles);
+		var erNåværendeInntektskategoriUdefinert = andelÅFordeleTil.getInputAndel().getInntektskategori().equals(Inntektskategori.UDEFINERT);
```

Comment:
Må ikkje du her sjekke at det ikkje allerede er opprettet ein fordelt andel? Vi vil ikkje opprette fra eksisterende for fleire enn ein inntektskategori.